src/main/java/com/thebois/Pawntastic.java

```diff
@@ -36,8 +37,9 @@
     private static final float VIEWPORT_HEIGHT = 1000;
     private static final int DEFAULT_FONT_SIZE = 26;
     private static final int PAWN_POSITIONS = 50;
-    private static final int TILE_SIZE = (int) Math.min(VIEWPORT_HEIGHT, VIEWPORT_WIDTH)
-                                         / WORLD_SIZE;
+    private static final int TOOLBAR_HEIGHT = 100;
```

Comment:
Only noticed this here first, but should `Pawntastic` know the toolbar (and other rendered elements) dimensions?

Reply:
Don't really know where to move though, since GameScreen, some views need the Toolbarheight, and most views and some controllers need the Tile size. 

I guess it's all about what do want everything to depend on, Application or some other class where we would move these.

Reply:
Shouldn't only `ToolbarView` need this height? `GameScreen` should just be able to get the height of the actor, no?

Reply:
Apparently, it was not needed anywhere only for calculating the tile size and indirectly the game view size. 
And then just have everything just expand and fill.